### DISCUSSION TITLE
Add Onlyfans?

### DISCUSSION THREAD

**Discussion by RamyCutz** [AAndyProgram/SCrawler#79] [Ideas]
Would you be able to add only fans to SCrawler or not something you interested in?

**Comment by AAndyProgram** [comment]
I can develop the OnlyFans downloader for SCrawler, but I don't want to pay for an account. I need a login and password for an account that contains at least one subscription. I need to see site responses to make a parser. No one gave me login/pass for the acc. A FREE subscription also requires a payment method. Sorry, but I will not add my card to the acc!

**Reply by Xtiantyr** [reply]
Yo te puedo brindar las cuentas gratis

**Reply by AAndyProgram** [reply]
Okay, message me on Discord.
AndyProgram#3804

**Reply by archwaycommons** [reply]
I wanted to check the status on this request. Was it ever implemented? Also do you have a discord for this program?

**Reply by AAndyProgram** [reply]
> Was it ever implemented?

No, and it won't until someone gives me a login/password with an active paid subscription. I don't pay for porn.

**Reply by archwaycommons** [reply]
Ok I can help with that. Do you have a way we can communicate outside of github?

**Reply by AAndyProgram** [reply]
Yes. My discord: AndyProgram#3804
But, unfortunately, I'm busy now and at least next week.

**Reply by Xtiantyr** [reply]
Yo te regalo cuentas

**Reply by AAndyProgram** [reply]
Added. Please update to the [latest release](https://github.com/AAndyProgram/SCrawler/releases/latest).

**Comment by Xtiantyr** [comment]
Donde puedo contctarte

**Reply by AAndyProgram** [reply]
1. Please, write in English.
2. I already [answered](https://github.com/AAndyProgram/SCrawler/discussions/79#discussioncomment-4174726) you! My discord: AndyProgram#3804

**Reply by Xtiantyr** [reply]
I can give you onlyfans accounts

**Reply by Xtiantyr** [reply]
xtiantyrr#8474

**Comment by Xtiantyr** [comment]
Write me this is my username xtiantyrr#8474

**Comment by Xtiantyr** [comment]
Write me this is my username xtiantyrr#8474
